prebid-mobile/prebid-mobile.md

```diff
@@ -11,31 +11,93 @@ nav_section: prebid-mobile
 
-Benefits and features include:
+Prebid Mobile helps app publishers by making the advantages of “traditional” web header bidding available to mobile app environments. This is accomplished by providing libraries that enable mobile app publishers to implement end-to-end, open source header bidding solutions.
 
-- Single integration point with Prebid Server with direct access to more mobile buyers
-- Server side configuration management for bidders and other settings, no need to update your app to make changes
-- Minimized latency due to background pre-caching of creatives
-- Increased yield due to increased competition between demand sources
-- A lightweight library that uses simple query strings to send ad requests and to pass results
+## Benefits and Features
 
-### How it Works
+Some of the benefits to using the Prebid Mobile header bidding solution include:
 
-The Prebid Mobile SDK sends requests to Prebid Server to help app developers to access additional demand that is not available from their mobile ad network.
+-   Provides a single integration point with Prebid Server, providing direct access to more mobile buyers.
+-   Server-side configuration management; no need for developers to update the app to make configuration changes.
+-   Prebid Mobile is a transparent, open source solution.
+-   Created for integration with other prebid products and open source servers.
+-   Flattens mediation layers, reducing ad ops burden for managing mediation partners.
+-   Latency is reduced compared to mediation.
 
-This means you **must** have a Prebid Server account in order to use Prebid Mobile. To set up your Prebid Server account for Prebid Mobile, refer to [Get Started with Prebid Server]({{site.github.url}}/prebid-mobile/prebid-mobile-pbs.html)
+## How It Works
 
- 1. As the Prebid Mobile module runs, it fetches bids from your demand partners integrated with Prebid Server.
- 2. Our Prebid Mobile module sends this bid information to your primary ad server SDK using key/value targeting.
- 3. The primary ad server SDK sends a request to the primary ad server including these custom parameters.  These parameters will match the key/value targeting previously configured on Prebid line items by your ad ops teams.
- 4. If the Prebid Mobile bid wins, the primary ad server returns our creative JS to your primary ad server SDK's ad view.
- 5. The creative JS in the ad view fetches the actual cached creative content from Prebid Server. When this happens, the impression is counted.
+Prebid Mobile works in conjunction with Prebid Server, demand partners, and your primary ad server to deliver programmable demand to your mobile app.
+
+### What You Need to Get Started
+
+Before you begin to use Prebid Mobile in your apps, you need to prepare your end-to-end system.
+
+**Mobile Developers**
+
+The following is a one-time setup process for *mobile developers*:
+
+-   [Set up a Prebid Server account]({{site.github.url}}/prebid-mobile/prebid-mobile-pbs.html).
+
```

Comment:
Should Prebid Server account set up be part of Ad Ops set up?

Reply:
@mjacobsonny ? Does ad ops set up prebid server, not the dev?

Reply:
Depends on the publisher's org. structure.  In most cases, ad ops will create the PBS account and the PBS configs.  Developers will include the PBS account ID and config ID(s) when integrating the Prebid Mobile SDK.